ui/watch.bat

```diff
@@ -0,0 +1,33 @@
+rmdir %~dp0node_modules /s /q
```

Comment:
Ohhh, this were my Windows compilation files. They were not supposed to be made public because most ppl are probably working with unix based systems. I am gonna remove them